plugins/src/webrtcsink/utils.rs

```diff
@@ -55,14 +55,10 @@ impl StreamProducer {
 
         gst_debug!(CAT, "Adding consumer");
 
-        consumer.set_property("max-buffers", 0u64).unwrap();
-        consumer.set_property("max-bytes", 0u64).unwrap();
-        consumer
-            .set_property("max-time", 500 * gst::ClockTime::MSECOND)
-            .unwrap();
-        consumer
-            .set_property_from_str("leaky-type", "downstream")
-            .unwrap();
+        consumer.set_property("max-buffers", 0u64);
```

Comment:
I was wondering when I ported another element, what was the rationale for removing the checks in the bindings? The fact that everyone ended up unwrapping anyway? What happens now if a property is the wrong type / doesn't exist? Silently fails as `gobject_set` would? I must say I appreciated not having to look up exact types everytime, and having the thing panic at runtime if I hadn't guessed correctly :)

Reply:
It will directly panic if you typoed the property name or used the wrong type. `set_property()` is basically `try_set_property().unwrap()` now (and if you want the old behaviour there's `try_set_property()`).

The reason for this change was that everybody `unwrap`ed anyway and it's more a programmer error (in 99% of the cases) than an error you are supposed to handle.

Reply:
Ah I see, thanks a lot